app/src/NativeBase/Components/Checkbox.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * @file CheckBox component for the STA Volunteer App
+ */
+
+import { Pressable, Icon } from 'native-base'
+import React from 'react'
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons'
+import StaTheme from '../Theme/StaTheme'
+type CheckboxProps = {
+  onChange: () => void
+  checked: boolean
+}
+
+/**
+ * Component that displays a checkbox that can be checked or unchecked.
+ *
+ * @param { CheckboxProps} props The component props
+ * @param {value: string[] => void} props.onChange function called when the checkbox is pressed
```

Comment:
Ideally this would be corrected to `() => void` (rather than `string[] => void`)

Reply:
Apologies, thought I had addressed this. Will get it changed. Cheers!